sunpy/coordinates/tests/test_utils.py

```diff
@@ -377,3 +379,33 @@ def test_limb_coords():
 
     with pytest.raises(ValueError, match='Observer distance must be greater than rsun'):
         get_limb_coordinates(observer, 1.1 * u.au)
+
+
+def test_get_heliocentric_angle():
```

Comment:
Break up this test. At the very least, separate the off disk ones and the ones that are testing an exception

Reply:
Broken up.